### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by rossmechanic**
Updating issue templates for bug reports and new feature requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (kseever) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Rename to `Environment`?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
```
Comment (kseever) [inline .github/ISSUE_TEMPLATE/feature_request.md]: How about a leading `Overivew` or `Problem Statement` section?

Comment (rossmechanic) [reply .github/ISSUE_TEMPLATE/feature_request.md]: What should we do as subsection/example there?

Comment (rossmechanic) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Isn't that pretty much what the question below is?

**Pull request merged**